public/docs/advanced/commands.md

```diff
@@ -1,129 +1,215 @@
 # Command reference
 
+This section will describe every command available in the Truffle application.
+
+
 ## Usage
 
-```none
-$ truffle [command] [options]
+All commands are in the following form:
+
+```shell
+truffle [command] [options]
 ```
 
+No arguments given is equivalent to `truffle help`, which will display a list of all commands and then exit.
+
+
 ## Available commands
 
+
 ### build
 
-Execute build pipeline (if configuration present)
+Execute build pipeline (if configuration present).
 
-```none
-$ truffle build
+```shell
+truffle build
 ```
 
-See the [Building your application](/docs/getting_started/build) section for more details.
+Requires the `build` key to be present in the configuration. See the [Building your application](/docs/getting_started/build) and [Build processes](/docs/advanced/build_processes) sections for more details.
```

Comment:
Not that this is in the scope of work, but I'd add a deprecating warning to this command. e.g., 

> This command is deprecated. Please use third-party build tools like webpack or grunt, or see our Truffle Boxes[LINKME] for an example.

Reply:
That's good to know. I wasn't sure which of these commands were ongoing concerns.